### ISSUE TITLE
Default email address

### ISSUE THREAD

**Issue by lorisadmin** [aces/Loris#4303]
The default email address is no-reply@example.com.  This causes problems with outgoing mail servers and is also likely to be flagged as spam.

The default should be no-reply@loris.ca or something like that.  Sites that do not use the loris.ca domain should use a no-reply@my-domain.com

**Comment by johnsaigle** [comment]
Notes from meeting:

Dave/Rida to look into php, replacing w Blank
* Raisinbread default will be updated to no-reply@loris.ca

MCIN-hosted instances (including Raisinbread installs): use no-reply@loris.ca 
* Externals : replace no-reply@example.com with your own… (as before)

**Comment by johnsaigle** [comment]
This is resolved.

**Comment by johnsaigle** [comment]
This is not resolved.

Raisinbread default email is still `admin@example.com`

**Comment by gdevenyi** [comment]
example.com is a reserved top level domains for exactly this application https://tools.ietf.org/html/rfc2606

**Comment by johnsaigle** [comment]
The issue is mostly in response to OP's comment 
> This causes problems with outgoing mail servers

This was written by our sysadmin who has since retired. Still, I believe the issue persists.

**Comment by johnsaigle** [comment]
> This is not resolved. Raisinbread default email is still admin@example.com

@ridz1208 Can you look into fixing this?

**Comment by christinerogers** [comment]
@ridz1208 should this be raised at the Loris meeting again in case @AlexandraLivadas can resolve it - or shall we keep this stale/close it?

**Comment by ridz1208** [comment]
I think the resolution is in the ticket itself. defualt should be changed to  no-reply@loris.ca

**Comment by driusan** [comment]
No, the default should be blank. See the thread. We should be setting a default to a real email address that goes to our domain.
